src/ManTradesTable.js

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import TradeRow from './TradeRow'
+import { PieChart } from 'react-minimal-pie-chart';
+
+
+import {
+    Button,
+    Table,
+    TableRow,
+    TableCell,
+    TextField,
+    TableBody,
+    TableHead,
+    Typography
+  } from "@material-ui/core";
+
+  import axios from "axios";
+  
+  let BASE_DOMAIN = `https://api.withlaguna.com`;
+  if (process.env.NODE_ENV === "development") BASE_DOMAIN = "http://0.0.0.0:5000";
+
+class ManTradesTable extends React.Component{
+    constructor(props){
+        super(props)
+        // const [trades, setTrades] = useState([])
+        this.state = {trades: [{placeholder: true}], total_weight: 0}
```

Comment:
I think design wise might make more sense to just have an empty row (rather than a placeholder) i.e. {ticker: '', weight: ''}.

Will make reduce function slightly easier below